examples/kserve_deployment/README.md

```diff
@@ -340,6 +340,14 @@ components are described in this document, but similar stacks may be set up
 using different backends and used to run the example as long as the basic Stack
 prerequisites are met.
 
+#### Deploying Individual Stack Components using the CLI
+
+The KServe model deployer and the GCP artifact store (i.e. as individual stack
+components) can be deployed using the ZenML CLI as well, using the `zenml
```

Comment:
"(i.e. as individual stack components)" what does this mean?

Reply:
You're right. Also unnecessary to say, actually. I removed it everywhere.